frontend-ssr/pages/_app.tsx

```diff
@@ -12,6 +13,16 @@ export default function MyApp({ Component, pageProps }: any) {
     },
   });
 
+  useEffect(() => {
+    if (process.env.NODE_ENV === 'production') {
+      const runLogRocket = async () => {
```

Comment:
I know TS doesn't allow async in `useEffect` but I think we should use Promises in this case. This might be confusing to someone who doesn't know  about this TS thing. In other words, at first view I might think: why is he creating a separate function and calling it.

Also, remember to handle the catch. A `console.error` is ok for now, we'll push those to the server at some point :D